### DISCUSSION TITLE
Transfer users from existing app to new app.

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#9566] [Help] [answered]
Is there a way to backup and restore or copy users from existing app to a new app. 
Example scenario i want to restore an exported app to create some new feature but also keep the old one and provide access to same set of users with same privileges'. Is there a way to do it? Export table is disabled for users, so I cannot export / import this information.

**Comment by melohagan** [comment, marked as answer]
Hey @gmentzik 

The easiest way to do this would be to use user groups. https://docs.budibase.com/docs/user-groups

That way when you restore the exported app you simply give that user group permission to the app!

**Reply by gmentzik** [reply]
We are using self host version without groups. So a method to backup and resusing same users with same rights at an imported or new app would be very convinient! Also I am thinking of same action for a clean installation of Budibase. So in clean installation import app and import also same set of users. Maybe i am  missing something on what hides behind the users table, but generally is very inconvenient for me to recreate the app users table every time I create a new app revision.
